client/src/components/all-components/HTMLComponents/Tab.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Tab = () => {
+  return <>&nbsp;&nbsp;&nbsp;</>;
```

Comment:
Wonder if we could make this a loop that adds `&nbsp;` based on the number of tabs you want, you could define a const for one tab (4 `&nbsp;`) and concat to a return value based on the number of tabs the user wants

Reply:
i agree

Reply:
Created functioning MultiTab component that user can specify number of tabs they want to be displayed.